docs/pages/access-controls/device-trust/enforcing-device-trust.mdx

```diff
@@ -41,30 +41,34 @@ can be configured with the `spec.options.device_trust_mode` option and
 applies to the resources in its `allow` rules. It
 works similarly to [`require_session_mfa`](../guides/per-session-mfa.mdx).
 
-<Admonition type="tip" title="v13.3.6+">
-  Teleport version 13.3.6 and above has the preset `require-trusted-device` role.
-  Make sure you update the "allow" rules in the role according to your requirements.
+<Admonition type="warning" title="require-trusted-device role">
+  The preset `require-trusted-device` role does not enforce
+  [Apps](#app-access-support) or [Desktops](#desktop-access-support).
+  Refer to their corresponding sections for instructions.
 </Admonition>
 
-To enforce authenticated device checks for a specific role, update the role with
-the following:
+The role below is an example of full device trust enforcement. Alternatively,
```

Comment:
I find it a bit confusing that the previous paragraph says "require-trusted-device does not do full enforcement" followed by an example role named "require-trusted-device" which _does_ do full enforcement.

Can we change the name of the role here?

Reply:
Changed the example to better match require-trusted-device.